examples/cppgc-object.rs

```diff
@@ -0,0 +1,127 @@
+// Copyright 2019-2021 the Deno authors. All rights reserved. MIT license.
+use std::cell::Cell;
+
+struct Wrappable {
+  id: String,
+  trace_count: Cell<u16>,
+}
+
+impl v8::cppgc::GarbageCollected for Wrappable {
+  fn trace(&self, _visitor: &v8::cppgc::Visitor) {
+    println!("Wrappable::trace() {}", self.id);
+    self.trace_count.set(self.trace_count.get() + 1);
+  }
+}
+
+impl Drop for Wrappable {
+  fn drop(&mut self) {
+    println!("Wrappable::drop() {}", self.id);
+  }
+}
+
+const DEFAULT_CPP_GC_EMBEDDER_ID: u16 = 0xde90;
```

Comment:
**suggestion:** Maybe add a comment here about where this comes from or how it's decided.